_studio/shared/src/libmfx_core_vaapi.cpp

```diff
@@ -41,11 +41,6 @@
 #include "va/va.h"
 #include <va/va_backend.h>
 
-#if (MFX_VERSION >= MFX_VERSION_NEXT)
```

Comment:
@onabiull : is that OK to delete this? recently I am afraid of any changes to versions stuff.

Reply:
Lines 44-47 is a dummy code. I just removed it

Reply:
got it, my mistake